builder/proxmox/iso/step_download_iso.go

```diff
@@ -0,0 +1,50 @@
+package proxmoxiso
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Telmate/proxmox-api-go/proxmox"
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+)
+
+// stepDownloadISO downloads an ISO file directly to the specified PVE node.
+// Checksums are also calculated and compared on the PVE node, not by Packer.
+type stepDownloadISO struct{}
```

Comment:
The code looks good here, I'm just a little bit worried the name may be confusing vs. the `StepDownload` that comes with the sdk, we could probably rename it to make it clearer that the download will be performed exclusively on the PVE server, and not through go-getter here.
Would `stepDownloadISOOnPVE` be OK to you?

Reply:
Sure, I renamed the step.